scripts/build.js

```diff
@@ -1,14 +1,19 @@
 const ncc = require("../src/index.js");
-const { statSync, writeFileSync, readFileSync, unlinkSync } = require("fs");
+const { statSync, writeFileSync, readFileSync, unlinkSync, rmdirSync } = require("fs");
 const { promisify } = require("util");
-const { relative } = require("path");
+const { relative, join } = require("path");
 const copy = promisify(require("copy"));
 const glob = promisify(require("glob"));
 const bytes = require("bytes");
 
 const minify = true;
+const v8cache = true;
+const cache = join(__dirname, "..", ".cache");
 
 async function main() {
+  if (process.argv[2] === "--no-cache") {
+    rmdirSync(cache, { recursive: true });
+  }
```

Comment:
Then `--no-cache` is just `cache: false` I believe.

Reply:
@guybedford What about `v8cache`, should that always be true?

Reply:
That creates a v8 snapshot to speedup the Node.js execution of ncc, so yes.